README.md

```diff
@@ -8,7 +8,54 @@ Hook that can be used for synchronous registration and deregistration in
 [Consul][1] discovery service on [Kubernetes][2] or [Mesos][3] cluster with 
 Allegro [executor][4].
 
+## Development
+
+To develop the hook locally you need the following things to be installed on 
+your machine:
+
+* [Minikube][5]
+* [Go][6]
+
+When everything is installed and setup properly, you can build hook for the Linux 
+operating system (as Minikube starts Kubernetes cluster on Linux virtual machine):
+
+```
+GOARCH=amd64 GOOS=linux go build -v ./cmd/consul-registration-hook
+```
+
+After successful build, you can start your local mini Kubernetes cluster:
+
+```
+minikube start
+```
+
+Mount project root to the Kubernetes virtual machine. You have to do it with the
+help of the `screen` utility because the process must be keep working to the 
+folder be properly mounted:
+
+```
+screen minikube mount .:/hooks
```

Comment:
Will it work with this? -> ```minikube start --mount --mount-string .:/hook```

Reply:
Yes, it will work - I will change the docs.